libc-top-half/musl/src/env/__init_tls.c

```diff
@@ -130,11 +130,9 @@ static struct builtin_tls
 #define MIN_TLS_ALIGN offsetof(struct builtin_tls, pt)
 
 static struct tls_module main_tls;
-#endif
```

Comment:
again, these changes are here by mistake

Reply:
This is actually on purpose, it enlarges the `__wasilibc_unmodified_upstream` ifdef to include the `__copy_tls` function which is unused.